Documentation/guideline/csharp_coding_standards.md

```diff
@@ -112,24 +118,24 @@ The class definition contains class members in the following order, from *less*
 ### Cross-platform coding
 Our code should supports multiple operating systems. Don't assume we only run (and develop) on Windows. Code should be sensitvie to the differences between OS's. Here are some specifics to consider.
 
-* **DO** use `Enviroment.NewLine` instead of hard-coding the line break instead of `\r\n`, as Windows uses `\r\n` and OSX/Linux uses `\n`.
+* **DO** use `Enviroment.NewLine` instead of hard-coding the line break, as Windows uses `\r\n` and OSX/Linux uses `\n`.
 
-> **Note**
->
-> Be aware that thes line-endings may cause problems in code when using `@""` text blocks with line breaks.
+  > **Note**
+  >
+  > Be aware that thes line-endings may cause problems in code when using `@""` text blocks with line breaks (it spans over multiple lines).
```

Comment:
(it spans over multiple lines) is not accurate. `@""` text blocks with line breaks may cause problems under different OS with different git crlf config. e.g. under Windows, files may be checked out with CRLF line endings, thus, the line breaks in `@""` change to `\r\n`.